source/Ship.cpp

```diff
@@ -1913,6 +1913,16 @@ double Ship::JumpFuel() const
 
 
 
+double Ship::JumpFuelMissing() const
+{
+	// Used for smart refuelling: transfer only as much as really needed
+	// includes checking if fuel cap is high enough at all
+	if(!JumpsRemaining() && attributes.Get("fuel capacity") > JumpFuel())
+		return JumpFuel() - fuel;
```

Comment:
I think that should be a `>=`, not a `>`. For example, if a ship's fuel capacity is 200 and it has a jump drive, as written this function would always return 0.